app/konnect/runtime-manager/plugins.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Plugins
+content_type: explanation
+---
```

Comment:
This is a good opportunity to include a screenshot of the UI, and a small and less detailed version of the chart here:
![image](https://user-images.githubusercontent.com/23319190/207971907-1aeedbc5-3e39-48d7-8a04-3d57a1d23413.png)

Reply:
https://docs.konghq.com/gateway/latest/kong-manager/#top-menu

This chart.